page/sass/invest-platform.scss

```diff
@@ -0,0 +1,24 @@
+.invest-platform {
+  h2 {
+    margin-top: 100px;
+    margin-bottom: 30px;
+    color: $gray-darker;
+    font-size: $section-title-font-size;
+    font-weight: $font-weight-more-bold;
+  }
+
+  .product-image {
+    margin: 0 auto;
+    max-height: 520px;
+
+    @media (max-width: $screen-md-min) {
```

Comment:
It should be `$screen-md-max`- https://www.mtech-llc.com/blog/david-paredes/how-use-bootstrap-media-query-breakpoints